src/os/nusys/nusched.c

```diff
@@ -3,21 +3,23 @@
 
 NOP_FIX
 
+NUSched nusched;
 NUScPreNMIFunc nuScPreNMIFunc = NULL;
+u8 nuScPreNMIFlag;
 
 char nusys_version[] = "NuSystem2.05";
 
-u32 nuScRetraceCounter = (u32) nusys_version; // wtf?
+u32 nuScRetraceCounter = (u32) nusys_version;
```

Comment:
does this make sense to you? I think I'd prefer we keep the comment here unless it has some reasonable explanation

Reply:
Yeah, but It's in the original NuSYS 2.05 code that it was actually forced intact and not a coincidence, as far my understanding goes. It's conflicting whether or not I should remove that comment intentionally, as I wanted to make it match the original code.

But you've actually noticed this so I will take action to remove this change now...

Reply:
Ah, I didn't realize it was like this in the actual code. In that case the comment can definitely be removed... Sorry to make you undo your change again